examples/Demo/Shared/Pages/Dialog/Examples/DialogServiceCallbackExample.razor

```diff
@@ -42,6 +42,10 @@
             {
                 Title = $"Hello {simplePerson.Firstname}",
                 OnDialogResult = DialogService.CreateDialogCallback(this, HandleDialog),
+                // OnDialogClosing = EventCallback.Factory.Create(this, async () =>
+                // {
+                //     await Task.Delay(1000);    
+                // }),
```

Comment:
Either include the code or remove it. Don't use commented out code.

Reply:
Thank you, I addressed this in the latest commit 201fa727268fee72a92202cec2fc4eaa0d9393b2